manual/Website1/package.json

```diff
@@ -6,9 +6,10 @@
     "url": "https://github.com/faceyspacey/remixx.git"
   },
   "author": "Zack Jackson <zack@ScriptedAlchemy.com> (https://github.com/ScriptedAlchemy)",
+  "license": "GPL-3.0-only",
```

Comment:
@ScriptedAlchemy is there any strong reason why `MIT` couldn't be the license for the package?

This will avoid hustle for some folks due to legal reasons from some organizations 😢

Reply:
Whats a better restrictive license.  I want people to be able to use it but want to restrict as much as possible.

Reply:
Moving license conversation to #107 - this PR really just copies the already existing used license string to the other `package.json`s so yarn stops complaining.